src/form/form-group/form-group.html

```diff
@@ -1,9 +1,9 @@
-<div class="form-group" ng-class="{ 'has-error' : hasErrors() }">
-  <label for="{{ pfField }}" class="control-label {{ pfLabelClass }}">{{ pfLabel }}</label>
-  <div class="{{ pfInputClass }}">
+<div class="form-group" ng-class="{ 'has-error' : $ctrl.hasErrors() }">
+  <label for="{{ $ctrl.pfField }}" class="control-label {{ $ctrl.pfLabelClass }}">{{ $ctrl.pfLabel }}</label>
```

Comment:
This is the example code. It shows both normal and horizontal forms:

![image](https://cloud.githubusercontent.com/assets/11633780/21398809/da123ae4-c776-11e6-8145-31778d747dc6.png)